### DISCUSSION TITLE
delete the isolation point

### DISCUSSION THREAD

**Discussion by Chrisable** [pyg-team/pytorch_geometric#5267] [General]
Do isolation points in data need to be deleted in GNN training？

**Comment by rusty1s** [comment]
Usually no, isolated nodes will simply not receive any messages. In general, it means that their computation graph is reduced to an MLP.

**Reply by Chrisable** [reply]
You mean deleting them will have no impact on the model

**Reply by rusty1s** [reply]
It can have an impact, if you apply a global readout layer you will either integrate the features of isolated nodes or not. However, during message passing, the presence of isolated nodes has no effect on other nodes.
